code/src/index.css

```diff
@@ -1,13 +1,119 @@
+
+
 body {
```

Comment:
One thing I think is good to get in the habit of, and that I am also working on for my self is to structure the CSS a bit. It easily gets long and difficult to overview. So maybe adding some comments and making the CSS file more divided for next project (at least if the file is long )